Frameworks/CoreGraphics/CGFont.mm

```diff
@@ -62,7 +62,9 @@ void CGFontSetFTFontSize(id uiFont, void* ftFont, float pointSize, float scale)
 }
 
 CGSize CGFontDrawGlyphsToContext(CGContextRef ctx, WORD* glyphs, DWORD length, float x, float y) {
-    return ctx->Backing()->CGFontDrawGlyphsToContext(glyphs, length, x, y);
```

Comment:
Good question. I know that for invalid conditions some of the layer and stack stuff aborts immediately.

---
In reply to: [83026505](https://github.com/Microsoft/WinObjC/pull/1126#discussion_r83026505) [](ancestors = 83026505)